include/jrl/IOValues.h

```diff
@@ -120,8 +120,8 @@ template <typename A1, typename A2, typename B = typename gtsam::Bearing<A1, A2>
 json serializeBearingRange(gtsam::BearingRange<A1, A2> bearingrange) {
   json output;
   output["type"] = BearingRangeTag;
-  output["bearing"] = serialize<B>(bearingrange.bearing());
-  output["range"] = serialize<R>(bearingrange.range());
+  output["bearing"] = io_values::serialize<B>(bearingrange.bearing());
```

Comment:
Since this is defined within the `io_values` namespace we don't need to specify namespace here. IMO we should not since it implies that these functions are coming from another namespace.

Reply:
Done!